### DISCUSSION TITLE
Blogging Series Support

### DISCUSSION THREAD

**Discussion by sheldonhull** [squidfunk/mkdocs-material#3395] [Ideas]
@squidfunk @eyllanesc To avoid dilution of the primary post on blogging #3353 , I'm responding here with some detail regarding series support for blogging here as it's not a core blogging feature, but an enhancement.

> It should also implement the concept of "Series", that is, a set of related articles. An example is [kubasejdak.com/techniques-of-variadic-templates](https://kubasejdak.com/techniques-of-variadic-templates):
> 
> ![Screenshot_20211017_170724](https://user-images.githubusercontent.com/4896200/137646660-5ffe65de-b417-4aa7-99fb-29f4e11fb825.png)

I am not as familiar with the templating engine of Mkdocs (Jinga?), but thought maybe the implementation I leveraged in Hugo would be useful.

I found the "partials" in Hugo templates solved this very nicely by allowing you to just add a front-matter value and have it show up based on matching series name. 

- [My Goland Log](https://www.sheldonhull.com/my-goland-log/) is flagged with a series title for Goland.
- The front matter just requires me to add `series: ['goland']` and whenever I rebuild my site all the pages get linked in publish order. There is no more having to maintain a "this is post n of x" like I did manually before.
- The rendering looks like image below
- And the partial is constructed like this: [series.html](https://github.com/sheldonhull/sheldonhull.hugo/blob/main/layouts/partials/series.html)

![Screenshot 2021-12-29 152508](https://user-images.githubusercontent.com/3526320/147704536-ee415ead-07cc-4c9e-bba0-c6a53145d4cc.png)

**Comment by eyllanesc** [comment]
[DoIt theme](https://github.com/HEIGE-PCloud/DoIt) also implements the feature.

**Comment by squidfunk** [comment]
Thanks, this is indeed a great idea! I'm not sure it will be part of the first iteration, but definitely something worth adding when we got basic blogging support of the ground.
